_security-analytics/threat-intelligence/api/source.md

```diff
@@ -0,0 +1,480 @@
+---
+layout: default
+title:  Source API
+parent: Threat intelligence APIs
+grand_parent: Threat intelligence
+nav_order: 50
+---
+
+# Source API
+
+The threat intelligence Source API updates and returns information about tasks related to threat intelligence source configurations.
+
+## Create or update threat intelligence source
+
+Creates or updates a threat intelligence source and loads Indicators of Compromise (IoCs) from that source.
+
+### Path and HTTP methods
+
+```json
+POST _plugins/_security_analytics/threat_intel/sources
+PUT _plugins/_security_analytics/threat_intel/sources/<source_id>
+```
+
+### Request Fields
+
+| Field  | Type  | Description  |
```

Comment:
The table is formatted incorrectly: 
<img width="663" alt="image" src="https://github.com/user-attachments/assets/e95d3a1a-d30b-4d0f-80e5-659e40d66cbd">

Reply:
Looks good!